packages/logging/jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    preset: "ts-jest",
+    testEnvironment: "node",
+    testMatch: ["**/test/**/*.test.ts"],
```

Comment:
let's make it like in other apps/packages pls
```suggestion
    testMatch: ["**/test/**/*.spec.ts"],
```